projects/js-packages/charts/src/components/line-chart/line-chart.tsx

```diff
@@ -110,12 +110,8 @@ const LineChart: FC< LineChartProps > = ( {
 		const seriesColors =
 			data?.map( series => series.options?.stroke ?? '' ).filter( Boolean ) ?? [];
 		return buildChartTheme( {
-			backgroundColor: providerTheme.backgroundColor,
+			...providerTheme,
```

Comment:
Nice one 🙌   I like how you've simplified the theme configuration by spreading the provider theme directly - much cleaner!